pkg/client/endpoint.go

```diff
@@ -49,27 +50,49 @@ func (r listResponse) MarshalJSON() ([]byte, error) {
 		})
 	}
 
+	sort.Sort(cs)
+
 	return json.Marshal(struct {
-		Clients []client `json:"clients"`
+		Clients responseClientList `json:"clients"`
 	}{
 		Clients: cs,
 	})
 }
 
+type responseClient struct {
+	CreatedAt time.Time `json:"created_at"`
+	Deleted   bool      `json:"deleted"`
+	ID        string    `json:"id"`
+	Name      string    `json:"name"`
+	Token     string    `json:"token"`
+}
+
+type responseClientList []responseClient
+
+func (l responseClientList) Len() int {
+	return len(l)
+}
+
+func (l responseClientList) Less(i, j int) bool {
```

Comment:
I thing `CreatedAfer` makes more sense

Reply:
This method is needed in this form to satisfy the [`sort.Interface`](https://godoc.org/sort#Interface) which we need to leverage [`sort.Sort`](https://godoc.org/sort#Sort) to always return the clients in the response in reverse chronological order.